tools/run.sh

```diff
@@ -46,5 +46,8 @@ done
 # Show success message once dev server is up
 listen_for_devserver &
 
+# Run Celery worker process
+celery -A integreat_cms.integreat_celery worker -l INFO -B --concurrency=1 --time-limit=3600 &
```

Comment:
`--time-limit=3600` will break all tasks running for longer than 1 hour. This should be sufficient for the statistics, right @PeterNerlich ?

Reply:
Why do we limit it to one hour and not four hours which has been defined above?